modules/eks/submodules/k8s/templates/aws-auth.yaml.tftpl

```diff
@@ -11,6 +11,7 @@ data:
       groups:
         - system:bootstrappers
         - system:nodes
+        %{ if nodes_master }- system:masters%{ endif }
 %{ endfor ~}
 %{ for role, arn in eks_master_role_arns ~}
```

Comment:
Could we pass eks_custom_role_maps through instead?

Reply:
we need the nodes arn which gets created in the module.